internal/sirius/add_payment.go

```diff
@@ -46,4 +46,4 @@ func (c *Client) AddPayment(ctx Context, caseID int, amount int, source string,
 	}
 
 	return nil
-}
+}
```

Comment:
Your IDE should probably be adding newlines to the end of files, and this one shouldn't have been removed.